sig-scheduling/annual-report-2024.md

```diff
@@ -12,36 +12,40 @@
    - Governance and leadership changes
 -->
 
+- **The performance improvement**
+  - Have put a lot of effort into QueueingHint ([KEP-4247](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/4247-queueinghint)), which enhances the scheduler's retrying efficiency.
```

Comment:
Maybe something like: Iterated on QueueingHints towards re-enabling the feature.